actionRuntimes/actionProxy/invoke.py

```diff
@@ -1,136 +0,0 @@
-#!/usr/bin/env python
```

Comment:
this script is linked to from a few blogs and is handy for testing containers directly.

Reply:
Okay I'll find another home for it.